src/components/node/Node.jsx

```diff
@@ -133,6 +134,10 @@ export default class Node extends React.Component {
             gtx -= tx;
             gty -= ty;
         } else {
+            if (typeof size === "object") {
+                console.warn("node.size should be a number when not using custom nodes.");
```

Comment:
Import `logWarning`: `import { logError } from "../../utils";`

```suggestion
                logWarning("node.size should be a number when not using custom nodes.");
```